courses/machine_learning/deepdive2/how_google_does_ml/solutions/intro_bqml_solution.ipynb

```diff
@@ -29,18 +29,32 @@
     "---"
```

Comment:
<p>Remove the objectives:</p><p> </p><ul><li>Use feature engineering to improve model accuracy</li><li>Predict baby weight from a set of features</li></ul><p> </p><p>(since per Lak we remove the More advanced and feature engineering section)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Objectives have been removed</p>